### DISCUSSION TITLE
export / print

### DISCUSSION THREAD

**Discussion by forenade** [kimai/kimai#2758] [Questions & Answers]
Hello i have a small edit question.
When iam gone export and print out my employs timesheets  i need to edit the SUMMERY and FULL LIST to get the list like i wont it to be. and that's not a big deal when you to it once but i need do do it alot.
Is there away to edit these so they are default like i wont it to right away when i come there ?
And can i edit the big text EXPORT OF TIMESHEETS to whatever i wont it to ?

Iam no server pro som please keep it a bit simple :)

**Comment by kevinpapst** [comment]
test the demo, I changed quite a bit in the print export (state saving, edit title ...)

**Reply by forenade** [reply]
I when in to the development version and it looks like that would fix my problems. So i try update the server to that version and see if that's suites me better. What is the easiest way to update it without losing all settings ?

Thanks for a great software by the way !

**Reply by kevinpapst** [reply]
Carefully read and apply the steps documented at: https://www.kimai.org/documentation/updates.html

**Reply by forenade** [reply]
sry if iam a noob but i cant get that update to work. Do i need to do a hole new setup now when i wont to go from 1.14.3 to development version?

**Reply by kevinpapst** [reply]
Please wait for the official release before updating, if you don't know how to do it now.
Working outside the documented way needs special IT skills and I cannot provide free support for that.

**Reply by forenade** [reply]
How far away is that? the new release?
I could just reinstall the hole server if the release is not coming soon or is it better to stick to the stable one?

**Reply by forenade** [reply]
i see the new realese is out now but when i follow the update documentation i get this errror when i use the comand git checkout 1.15.1


error: Your local changes to the following files would be overwritten by checkout:
        symfony.lock
Please commit your changes or stash them before you switch branches.


what am i missing ?

**Reply by kevinpapst** [reply]
Try `git reset --hard`, followed by the normal update procedure.

**Reply by forenade** [reply]
that fix my problem. big thanks for great update
